portality/forms/application_forms.py

```diff
@@ -1815,6 +1835,26 @@ class FieldDefinitions:
         ],
         "help" : {
             "render_error_box": False
+        },
+        "contexts": {
+            "admin": {
+                "widgets": [
+                    "click_to_copy",  # ~~^-> ClickToCopy:FormWidget~~
+                    "full_contents"  # ~~^->FullContents:FormWidget~~
+                ]
+            },
+            "editor": {
+                "widgets": [
+                    "click_to_copy",  # ~~^-> ClickToCopy:FormWidget~~
+                    "full_contents"  # ~~^->FullContents:FormWidget~~
+                ]
+            },
+            "associate_editor": {
+                "widgets": [
+                    "click_to_copy",  # ~~^-> ClickToCopy:FormWidget~~
+                    "full_contents"  # ~~^->FullContents:FormWidget~~
+                ]
+            }
```

Comment:
since every role have the same widgets
can we remove `contexts` and change it to 
```
        "widgets" : [
            "tagentry",
             "click_to_copy",  # ~~^-> ClickToCopy:FormWidget~~
             "full_contents"  # ~~^->FullContents:FormWidget~~
        ],
```